modules/darwin/common/default.nix

```diff
@@ -58,15 +57,11 @@ in
   # disable application layer firewall, telegraf needs an incoming connection
   system.defaults.alf.globalstate = 0;
 
-  # srvos
-  services.openssh.authorizedKeysFiles = pkgs.lib.mkForce [ "/etc/ssh/authorized_keys.d/%u" ];
-
   # srvos
   environment.etc."ssh/sshd_config.d/darwin.conf".text = ''
     HostKey /etc/ssh/ssh_host_ed25519_key
```

Comment:
```suggestion
  environment.etc."ssh/sshd_config.d/darwin.conf".text = ''
    AuthorizedKeysFile none
    HostKey /etc/ssh/ssh_host_ed25519_key
```

Reply:
We may need to reinstall or switch the community builder to a different machine so easier to just require that the keys are managed.

Reply:
Makes sense, I forgot about the warning about `$HOME` :)